src/settings.cpp

```diff
@@ -34,3 +34,28 @@ void Settings::setPrettifySymbols(bool prettifySymbols)
         emit prettifySymbolsChanged(m_prettifySymbols);
     }
 }
+
+void Settings::setColorScheme(Settings::ColorScheme scheme)
```

Comment:
why is there a default and a non-default scheme? only one should suffice?

Reply:
The default scheme is selected when hotspot is started. The other one is the one the user selected. Maybe this is a bit overkill.